circle.yml

```diff
@@ -3,10 +3,10 @@ machine:
     version: 2.3.1
   environment:
     PR_NUMBER: $(echo ${CI_PULL_REQUEST##*/})
-    BUCKET_NAME: "site-$CIRCLE_BRANCH"
+    FOLDER_NAME: "site-$CIRCLE_BRANCH"
     WEBSITE_INDEX_DOCUMENT: index.html
     AWS_REGION: eu-west-1
-    PREVIEW_URL: "https://${BUCKET_NAME}.s3.amazonaws.com/$WEBSITE_INDEX_DOCUMENT"
+    PREVIEW_URL: "https://codurance-site-pr.s3.amazonaws.com/$FOLDER_NAME/$WEBSITE_INDEX_DOCUMENT"
```

Comment:
why not make `codurance-site-pr` into an env variable, just like $FOLDER_NAME, helps see the constant parts in the URL from the variable/substitution parts